src/main/java/online/shop/service/lib/Dao.java

```diff
@@ -0,0 +1,8 @@
+package online.shop.service.lib;
```

Comment:
```suggestion
package online.shop.lib;
```
`lib` is a separate package